ui/DesignSystem/Component/Transaction.svelte

```diff
@@ -36,35 +36,42 @@
   const summary = costSummary(
     transaction.messages[0].type,
     parseInt(transaction.fee),
-    transactionDeposits
+    registrationFee
   );
 
+  const txFeeRowPosition = summary.registrationFee ? "middle" : "top";
+
   $: updateAvatar();
 </script>
 
 <Header {transaction} {avatar} {subject} />
 
-<Row dataCy="deposit" variant="top" style="">
-  <div slot="left">
-    <Text variant="regular" style="color:var(--color-foreground-level-6);">
-      {formatStake(transaction.messages[0])}
-    </Text>
-  </div>
+{#if summary.registrationFee}
+  <Row dataCy="registration-fee" variant="top" style="">
+    <div slot="left">
+      <Text variant="regular" style="color:var(--color-foreground-level-6);">
+        {formatStake(transaction.messages[0])}
+      </Text>
+    </div>
 
-  <div slot="right">
-    <Rad rad={summary.depositRad} usd={summary.depositUsd} variant="deposit" />
-  </div>
-</Row>
+    <div slot="right">
+      <Rad
+        rad={summary.registrationFee.rad}
+        usd={summary.registrationFee.usd}
+        variant="deposit" />
```

Comment:
Should we rename the `variant` now?

Reply:
hm I don't think so. This refers to the field representing a credit or a deposit.

Reply:
Yes, we should remove it from the `<Rad />` component. And I think I'd add a total to the transaction view now, like this:
![section amounts](https://user-images.githubusercontent.com/2326909/88173430-de5fdc00-cc22-11ea-85b4-a014651a7ef1.png)

Reply:
@brandonhaslegs what do you think?

Reply:
I can drop the variants, @juliendonck. Could you address the UI change in a separate PR?

Reply:
sure, if you create an issue so I don't forget.

Reply:
just did it all :+1: